src/cmdc_tools/datasets/official/DE/data.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import requests
+import us
+
+from ...base import DatasetBaseNoDate
+from ..base import CountyData
+
+
+class Delaware(DatasetBaseNoDate, CountyData):
+    state_fips = int(us.states.lookup("Delaware").fips)
+    has_fips: bool = False
+    source_map = {
+        "Sussex": "https://myhealthycommunity.dhss.delaware.gov/locations/county-sussex/download_covid_19_data",
+        "Kent": "https://myhealthycommunity.dhss.delaware.gov/locations/county-kent/download_covid_19_data",
+        "New Castle": "https://myhealthycommunity.dhss.delaware.gov/locations/county-new-castle/download_covid_19_data",
+    }
+    source = list(source_map.values())
+
+    def get(self):
+        dfs = []
+        for (county, url) in self.source_map.items():
+            df = self._get_from_source(url)
+            df["county"] = county
+            dfs.append(df)
+
+        return pd.concat(dfs, sort=False).melt(
+            id_vars=["dt", "county"], var_name="variable_name"
+        )
+
+    def _get_from_source(self, source):
+        df = pd.read_csv(source)
+
+        df["dt"] = pd.to_datetime(df[["Year", "Month", "Day"]])
+        keep = df.loc[df.Unit == "people"][
+            ["Location", "County", "dt", "Statistic", "Value"]
+        ]
+        pivoted = (
+            keep.set_index(["Location", "County", "dt"])
+            .pivot(columns="Statistic")["Value"]
+            .reset_index()
+        )
+        renamed = pivoted.rename(
+            columns={
+                "County": "county",
+                "Confirmed Deaths": "deaths_confirmed",
+                "Cumulative Number of Confirmed Positive Cases": "cases_confirmed",
+                # 'Cumulative Number of Positive Cases': "cases_suspected",
+                "Deaths": "deaths_total",
+                "Cumulative Number of Probable Positive Cases": "cases_suspected",
+                "Recovered": "recovered_total",
+                "Tested Negative": "negative_tests_total",
+                "Total Persons Tested": "tests_total",
```

Comment:
This should be commented out for now because we don't have a `tests_total` variable in the database yet.

Reply:
Commented out